x/crisis/types/keys.go

```diff
@@ -3,4 +3,10 @@ package types
 const (
 	// module name
 	ModuleName = "crisis"
+
+	StoreKey = ModuleName
+)
+
+var (
+	ConstantFee = []byte("ConstantFee")
```

Comment:
Can we use a byte here instead?
```suggestion
	ConstantFeeKey = []byte("ConstantFee")
```